charts/astronomer/values.yaml

```diff
@@ -23,7 +23,7 @@ images:
     pullPolicy: IfNotPresent
   houston:
     repository: astronomerinc/ap-houston-api
-    tag: 0.15.9
+    tag: master
```

Comment:
@sjmiller609 @schnie I changed this to master because the houston-api version that is using yarn is not tagged.  I figured we can replace this with the correct 0.16.x version when that is ready.  Is that all right with you two? CC: @andriisoldatenko

Reply:
discussing houston image release process in slack